sauna/plugins/ext/memcached.py

```diff
@@ -0,0 +1,97 @@
+import re
+import socket
+
+from sauna.plugins import (Plugin, bytes_to_human, human_to_bytes,
+                           PluginRegister)
+
+my_plugin = PluginRegister('Memcached')
+
+
+@my_plugin.plugin()
+class Memcached(Plugin):
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.config = {
+            'host': config.get('host', 'localhost'),
+            'port': config.get('port', 11211),
+            'timeout': config.get('timeout', 5)
+        }
+        self._stats = None
+
+    @my_plugin.check()
+    def accepting_connections(self, check_config):
+        if self.stats['accepting_conns'] == 1:
+            return Plugin.STATUS_OK, 'Memcached is accepting connections'
+        else:
+            return Plugin.STATUS_CRIT, 'Memcached is not accepting connections'
+
+    @my_plugin.check()
+    def bytes(self, check_config):
```

Comment:
Can you add an other check like this one but with % of ram ? You can have this information with "limit_maxbytes" in stats